reddit/reddit.py

```diff
@@ -4,45 +4,55 @@
 import sys
 import time
 import re
-from pathlib import Path
 import praw.models
-from reddit_resource import subreddits as subs
+from datetime import datetime
+from pathlib import Path
+from reddit_resources import reddit_config as config
 from reddit_service import praw_service as p
-script_dir = os.path.dirname(os.path.abspath(__file__))
-crawl_scrape_dir = os.path.abspath(os.path.join(script_dir, '..'))
-sys.path.append(crawl_scrape_dir)
-from stock.stock_search import find_stock
-
-logging.basicConfig(level=logging.DEBUG)
+from reddit_service import logger as log
 
-cli_arguments = sys.argv[1:]
-
-def top_25_scanner():
+def subreddit_scanner():
     reddit = p.praw_connection()
-    for subreddit in subs.subreddit_list():
-        top25 = reddit.subreddit(subreddit).hot(limit=25)
-        top25Submissions = []
-        for submission in top25:
+    reddit_config = config.getConfig()
+    log.formatter()
+    for config_data in reddit_config["Subreddits"]:
+        subreddit = config_data["Name"]
+        if not subreddit:
+            continue
+
+        logging.info("Scanning subreddit: " + subreddit)
+        front_page_comments = []
+        try:
+            front_page = reddit.subreddit(subreddit).hot(limit=25)
```

Comment:
Thoughts on making the limit an env variable, but maybe setting the cap to 25 for now?

Reply:
all changes done, can be viewed here:

https://github.com/Sentmint/Crawl-Scrape/pull/20/commits/a175ef8ea8daf807ef2686b66e8787d08cebfb33